test/e2e/helpers.js

```diff
@@ -107,12 +106,7 @@ async function withFixtures(options, testSuite) {
     }
     const mockedEndpoint = await setupMocking(mockServer, testSpecificMock);
     await mockServer.start(8000);
-    if (
-      process.env.SELENIUM_BROWSER === 'chrome' &&
-      process.env.CI === 'true'
-    ) {
-      await ensureXServerIsRunning();
```

Comment:
We had found that the e2e tests would intermittently fail unless this was running. This took _a long time_ for us to debug. Eventually we settled on this solution after finding that the Chromium project uses a similar strategy for their own internal test suite.

I wonder why it's not required anymore... hopefully this won't bring the intermittent failures back.

Reply:
I'm not sure.. None of the Chrome E2E tests would start when this was being run with it. It seemed like it would never resolve 🤔 

Perhaps `xset` is not available on the new image? It's curious that it didn't throw though.

Reply:
It is only enabled in CI though 🤔

Reply:
Ah, interesting. That condition was not present in [the original PR](https://github.com/MetaMask/metamask-extension/pull/11887) where this was introduced.

Reply:
I would say we remove it for now and we can investigate further if it turns out to introduce intermittent test failures?